src/main/java/org/folio/clients/SearchClient.java

```diff
@@ -75,8 +78,13 @@ public SearchClient(WebClient webClient, ErrorLogService errorLogService) {
         } else {
           try {
             JsonObject instances = response.bodyAsJsonObject();
-            if (instances != null) {
-              promise.complete(Optional.of(instances));
+            if (nonNull(instances)) {
+              if (instances.getJsonArray("ids").isEmpty()) {
```

Comment:
Is it make sense to save "ids" as string constant?

Reply:
Good catch, fixed